src/SprykerSdk/Zed/ComposerConstrainer/Communication/Console/ComposerConstraintConsole.php

```diff
@@ -21,6 +21,8 @@ class ComposerConstraintConsole extends Console
     public const COMMAND_NAME = 'code:constraint:modules';
     public const OPTION_DRY_RUN = 'dry-run';
     public const OPTION_DRY_RUN_SHORT = 'd';
+    public const OPTION_WITH_FOREIGN = 'with-foreign';
+    public const OPTION_WITH_FOREIGN_SHORT = 'w';
```

Comment:
Mb smth like `wf`?

Reply:
According to [Option conventions in Unix-like systems](https://en.wikipedia.org/wiki/Command-line_interface#Option_conventions_in_Unix-like_systems) 

>  if one hyphen is followed by two or more letters it may mean two options are being specified, or it may mean the second and subsequent letters are a parameter (such as filename or date) for the first option.